_data/glossary/cloud-provider.yaml

```diff
@@ -0,0 +1,10 @@
+id: cloud-provider
+name: Cloud Provider
+full-link: /docs/reference/kubectl/docker-cli-to-kubectl/
```

Comment:
this link doesn't have the right underlying details for anything about cloud providers - copy/paste error? Perhaps a better one would be:

> `/docs/concepts/cluster-administration/cloud-providers/`